internal/raft/append_entries_test.go

```diff
@@ -0,0 +1,89 @@
+package raft
+
+import (
+	"net"
+	"testing"
+
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/assert"
+	"github.com/tomarrell/lbadd/internal/network"
+	"github.com/tomarrell/lbadd/internal/raft/cluster"
+	"github.com/tomarrell/lbadd/internal/raft/message"
+)
+
+// TestAppendEntries function checks the correctnes of AppendEntriesResponse
+// function. In this test function, we check how the function will respond to
+// if node Term is less than leader node, node Log Index is less than leader
+// commit Index and checks if logs are appended correctly to node Log
```

Comment:
```suggestion
// commit Index and checks if logs are appended correctly to node Log.
```

Godoc comments should be full sentences.

Reply:
Also its `commitIndex`